src/modules/routes/task.js

```diff
@@ -0,0 +1,25 @@
+const express = require('express');
+const router = express.Router();
+const taskValidator = require('../middleware/validate-middleware');
+const {
+  isCheckValidator,
+  textValidator
+} = require('../middleware/register-schema');
+
+const {
+  getAllTasks,
+  createNewTask,
+  changeTaskInfo,
+  changeTaskComplete,
+  deleteTask,
+  deleteAllTask
+} = require('../controllers/task.controller');
+
+router.get('/allTasks', getAllTasks);
```

Comment:
Route names must be from the REST API
for example 
`router.get('/tasks', getAllTasks);`